docs/UserGuide.md

```diff
@@ -65,17 +67,53 @@ command to run the application.<br>
+  - ![excessive fields](excessive_fields)
+- For `t/TAG, you may input both `t/BUYER` and/or `t/SELLER` (case insensitive). 
+  - If either tag is repeated more than once, for e.g. `t/BUYER t/BUYER`, the tag will only be recorded once and no error will be thrown. 
+- Note that `REMARK` and `BIRTHDAY` fields are optional, enclosed in `[]`.
+  - You do not need to include the prefixes for `r/REMARK` and `b/BIRTHDAY` if you do not wish to include them.
+  - But if you do include the prefix with only a blank string, birthday and remark fields will be taken as not specified.
+    - Example: `add n/John Doe p/98765432 i/20000 e/johnd@example.com a/311, Clementi Ave 2, #02-25 f/4 t/Buyer h/HDB r/ b/` will successfully add John Doe but remarks and birthday will be not specified. 
+- You will find the specific formats required for each field as shown below.
+  - If any of the formats are violated, you will receive an error message detailing the fields with invalid formats.
+  - For example, if name, phone and tag fields are in incorrect format, you will receive 3 error messages as shown.
+  - ![Error parsing fields](parsing_errors)
+- You may input the parameters in any order (e.g. if the command specifies `n/NAME a/ADDRESS`, `a/ADDRESS n/NAME` is also acceptable).
+- You may not have duplicate persons with the same name in realodex.
+  - If you try to add duplicate persons, you will get the error message "This client already exists in Realodex".
+  ![duplicate person](duplicate_person_error.png)
+
+Specific Formats for each field:
+* NAME: Should only contain Alphanumeric characters
+* PHONE: Should only contain numbers, and it should be at least 3 digits long
+* INCOME: Income should be an integer and should be at least 0
+* EMAIL: Emails should be of the format local-part@domain and adhere to the following constraints: 
+  1. The local-part should only contain alphanumeric characters and these special characters, excluding the parentheses, (+_.-). The local-part may not start or end with any special characters. 
+  2. This is followed by a '@' and then a domain name. The domain name is made up of domain labels separated by periods.\
+     The domain name must:
+     * end with a domain label at least 2 characters long 
+     * have each domain label start and end with alphanumeric characters 
+     * have each domain label consist of alphanumeric characters, separated only by hyphens, if any.
+* ADDRESS: Must not contain other prefixes within the address. (eg. `a/lemontree stree t/1`)
+* FAMILY: Should be an integer greater than 1.
+* TAG: only accept "buyer" or "seller" as the input (case-insensitive). Multiple tags are accepted (eg. `t/buyer t/seller`)
+* HOUSINGTYPE: must be one of the following: "HDB", "CONDOMINIUM", "LANDED PROPERTY", "GOOD CLASS BUNGALOW" (case-insensitive)'
+* REMARK: Should only contain Alphanumeric characters, and can be empty
+* BIRTHDAY: Should be in the form "DDMMMYYYY", and can be empty if the birthday is not specified. 
```

Comment:
could consider specifying that month in alphabetical not numerical

Reply:
Yes, I think can include a example for each field as well